geonode_mapstore_client/client/themes/geonode/less/_filter-form.less

```diff
@@ -81,13 +113,66 @@
     margin-left: auto;
 }
 
-.gn-sub-filter-items.checkbox {
-    
-    &:first-of-type {
-        margin-top: 5px;
+.gn-sub-filter-items {
+    .checkbox {
+
+        &:first-of-type {
+            margin-top: 0.3rem;
+        }
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+    }
+    .facet {
+        margin: 0.3rem;
+        margin-left: 1rem;
+        cursor: pointer;
+        &.active {
+            > span {
+                font-weight: bold;
+            }
+        }
+        span {
+            margin-right: 0.25rem;
+        }
     }
+}
 
-    &:last-child {
-        margin-bottom: 0;
+.gn-accordion {
+    .accordion-title {
+        display: flex;
+        align-items: center;
+        padding: 0.25rem;
+        border-width: 0.08rem;
+        border-style: solid;
+        border-radius: 0.25rem;
+        button {
+            border: none;
+            background: none;
+            box-shadow: none;
+            &:active:focus {
+                background: none;
+            }
+        }
+    }
+    .accordion-body {
+        padding: 0.6rem;
+        .accordion-items {
+            display: flex;
+            flex-direction: column;
+            gap: 0.3rem;
+            .facet {
+                cursor: pointer;
+                &.active {
+                    > span {
+                        font-weight: bold;
+                    }
+                }
+                span {
+                    margin-right: 0.25rem;
+                }
+            }
```

Comment:
the facet class should be applied independently from its own container, we could use the  accordion-items selector to add specific changes to the children such as:
```

.gn-accordion {
    .accordion-title {
        display: flex;
        align-items: center;
        padding: 0.25rem;
        border-width: 0.08rem;
        border-style: solid;
        border-radius: 0.25rem;
        button {
            border: none;
            background: none;
            box-shadow: none;
            &:active:focus {
                background: none;
            }
        }
    }
    .accordion-body {
        .accordion-items {
            display: flex;
            flex-direction: column;
            .facet {
                padding-left: 0.6rem;
            }
        }
    }
}
```